src/inmem/mutable.rs

```diff
@@ -348,4 +341,55 @@ mod tests {
             &Timestamped::new("4".into(), 0_u32.into())
         );
     }
+
+    #[tokio::test]
+    async fn test_dyn_read() {
+        let temp_dir = tempfile::tempdir().unwrap();
+        let option = DbOption::with_path(
+            Path::from_filesystem_path(temp_dir.path()).unwrap(),
+            "age".to_string(),
+            0,
+        );
+        let fs = Arc::new(TokioFs) as Arc<dyn DynFs>;
+        fs.create_dir_all(&option.wal_dir_path()).await.unwrap();
+
+        let trigger = Arc::new(TriggerFactory::create(option.trigger_type));
+
+        let mutable = Mutable::<DynRecord>::new(&option, trigger, &fs)
+            .await
+            .unwrap();
+
+        mutable
+            .insert(
+                LogType::Full,
+                DynRecord::new(
+                    vec![
+                        Column::new(Datatype::Int8, "age".to_string(), Arc::new(1_i8), false),
+                        Column::new(
+                            Datatype::Int16,
+                            "height".to_string(),
+                            Arc::new(1236_i16),
+                            true,
+                        ),
+                    ],
+                    0,
+                ),
+                0_u32.into(),
+            )
+            .await
+            .unwrap();
+
+        {
+            let mut scan = mutable.scan((Bound::Unbounded, Bound::Unbounded), 0_u32.into());
+            let entry = scan.next().unwrap();
+            assert_eq!(
+                entry.key(),
+                &Timestamped::new(
+                    Column::new(Datatype::Int8, "age".to_string(), Arc::new(1_i8), false),
```

Comment:
why is it just the value of one column in a record? Before, each mutable entry should be a record?